mobile-app/app/screens/TransactionAuthorization/TransactionAuthorization.tsx

```diff
@@ -43,7 +51,7 @@ let PASSPHRASE_PROMISE_PROXY: {
 const INVALID_HASH = 'invalid hash'
 const USER_CANCELED = 'USER_CANCELED'
 
-export type Status = 'INIT' | 'IDLE' | 'BLOCK' | 'PIN' | 'SIGNING'
+export type Status = 'INIT' | 'IDLE' | 'BLOCK' | 'PIN' | 'SIGNING' | 'AUTHORIZED' | 'ACCESS_GRANTED'
```

Comment:
Possible to combine these two new status? They seems to be serving same purpose.

Reply:
Sure. The `transaction` will be used instead as identifier to determine which completion message to display. Updated!